attributes/cacher.rb

```diff
@@ -0,0 +1,2 @@
+default['ssh_known_hosts']['cacher']['data_bag'] = 'server_data'
+default['ssh_known_hosts']['cacher']['data_bag_item'] = 'known_hosts'
```

Comment:
I'd prefer one file that was organized well, rather than a dozen files with an attribute or two each so I only have one file to open up in an editor.

And splitting things up into different files doesn't address separation of concerns.  It just means you have to load more files into the editor and context switch between them.